tutorials/intro-to-geospatial-analysis/index.md

```diff
@@ -0,0 +1,449 @@
+var buffGeo = geometry.buffer(100);
+```
+### Find bounded rectangle of the Geometry
+```javascript
+var bounGeo = geometry.bounds();
+```
+### Find the smallest envelope that can envelop the Geometry
+```javascript
+var convexGeo = geometry.convexHull();
+```
+### Find common area between two or more geometries
+```javascript
+var interGeo = geometry1.intersection(geometry2);
+```
+### Find area that includes two or more geometries
+```javascript
+var unGeo = geometry1.union(geometry2);
+```
+---
+## Operations on Features
+
+[Feature operations](https://code.earthengine.google.com/bb1980ff4b6ddab0262de00956f89dcb "EE Feature operations")
+
+### Set property name and value of geometry to create a feature
+```javascript
+var feat = ee.Feature(geometry, {Name: "featureName", Size: 500};
+```
+### Create a new feature from existing feature while renaming a property
+```javascript
+var featNew = feature.select(["name"], ["descriptor"]);
+```
+### Extract values of a property from a Feature
+```javascript
+var featVal = feature.get("size");
+```
+---
+
+## Filters
+
+### Creator a filter for values of a property
+
+```javascript
+var bFilter = ee.Filter.eq(propertyName, value);
+```
+>or .neq , .gt , .gte , .lt , and .lte
+
+### Create a filter based on maximum difference from a threshold
+
+```javascript
+var diffFilter = ee.Filter.maxDifference(threshold, propertyName, value);
+```
+
+### Create a text filter
+
+```javascript
+var txtFilter = ee.Filter.stringContains(propertyName, stringValue);
+```
+
+>or .stringStartsWith, and .stringEndsWith
+
+### Create a range filter
+
+```javascript
+var rangeFilter = ee.Filter.rangeContains(propertyName, stringValue, minValue, maxValue);
+```
+  
+### Create a list filter to check for certain values
+
+```javascript
+var listFilter = ee.Filter.listContains(propertyName, value1, propertyName2, value2);
+```
+>.inList to test against list of values
+
+### Create a filter of dates
+
+```javascript
+var dateFilter = ee.Filter.calendarRange(startDate, stopDate);
+```
+
+### Create a filter for particular days of the year
+
+```javascript
+var dayFilter = ee.Filter.dayOfYear(startDay, stopDay);
+```
+
+### Create a filter to subset geospatial data
+
+```javascript
+var boundsFilter = ee.Filter.bounds(geometryOrFeature);
+```
+### Combining and inversing filters
+
+```javascript
+var newFilter = ee.Filter.and(listOfFilters);
+var newFilter = ee.Filter.or(listOfFilters);
+var inverseFilter = ee.Filter.not(filter);
+```
+---
+## Operations on Images
+
+[Image operations](https://code.earthengine.google.com/234ba0ab11dd82112602de05c0bc3bd8 "EE Image operations")
+
+### Selecting the bands of an image
+```javascript
+var band = var_Image.select(bandName);
+```
+### Creating masks
+```javascript
+var mask = image.eq(value);
+```
+>or .neq or .gt or .gte or .lt or .lte
+
+### Applying masks
+```javascript
+var masked = image.mask(mask);
+```
+
+### Pixelwise calculation
+```javascript
+var results = image.sum(value);
```

Comment:
I think this should be image.add() (not sum).

Reply:
Thanks! Fixed it.